### PR TITLE
Meeting Calls / Reporting by RPC

### PR DESCRIPTION
**Pull request by manianac**
Multiplayer/host requirement is no longer needed

### CODE REVIEW

File: gui/tabs/players_tab.cpp [hunk 1]
```diff
@@ -41,44 +41,42 @@ namespace PlayersTab {
...
-							ImGui::NewLine();
-							if (State.selectedPlayerId > -1 && State.selectedPlayer != *Game::pLocalPlayer && ImGui::Button("Report Body")) {
-								PlayerControl_CmdReportDeadBody(*Game::pLocalPlayer, GetPlayerData(State.selectedPlayer), NULL);
-							}
+					if (!GetPlayerData(*Game::pLocalPlayer)->fields.IsDead) {
+						if (ImGui::Button("Call Meeting")) {
+							State.rpcQueue.push(new RpcReportPlayer(*Game::pLocalPlayer, NULL));
+						}
+						ImGui::NewLine();
+						if (State.selectedPlayerId > -1 && State.selectedPlayer != *Game::pLocalPlayer && ImGui::Button("Report Body")) {
```
Comment (dencelle) [inline gui/tabs/players_tab.cpp:49]: need to make sure that the selected player that we are attempting to report is dead.

File: gui/tabs/players_tab.cpp [hunk 2]
```diff
@@ -41,44 +41,42 @@ namespace PlayersTab {
...
+							State.rpcQueue.push(new RpcReportPlayer(*Game::pLocalPlayer, State.selectedPlayer));
 						}
+					}
 
-						if (State.selectedPlayerId > -1 && !GetPlayerData(State.selectedPlayer)->fields.Disconnected && State.selectedPlayer != *Game::pLocalPlayer)
+					if (State.selectedPlayerId > -1 && !GetPlayerData(State.selectedPlayer)->fields.Disconnected && State.selectedPlayer != *Game::pLocalPlayer)
```
Comment (dencelle) [inline gui/tabs/players_tab.cpp:54]: need a check to make sure we don't spectate a ghost while we are alive. something tells me that would look very odd to the server. i could be wrong though.

**Review by dencelle — CHANGES_REQUESTED** [review]
just recommending some small sanity checks. i know they aren't related to what you're actually changing but if we are pushing this through, might as well change them now anyways.

how sure are we that removing the muliplayer and host checks won't lead to people getting banned?

**Comment by std-nullptr** [comment]
It looks like there are conflicts as the main branch was squashed and rebased, a new branch will need to be created and this will have to be redone. Sorry :(

File: gui/tabs/players_tab.cpp (hunk 2 above)
Comment (manianac) [reply gui/tabs/players_tab.cpp:54]: > just recommending some small sanity checks. i know they aren't related to what you're actually changing but if we are pushing this through, might as well change them now anyways.
> 
> how sure are we that removing the muliplayer and host checks won't lead to people getting banned?

I've been playing for about a week now, test calling random meetings and reporting bodies.  Only bans I get is if I call a meeting or report as another player.  As long as you call from pLocalPlayer you're good.

Comment (KyreusR) [reply gui/tabs/players_tab.cpp:54]: Please make a new PR with our rebased main branch and I'll test it.

**Pull request closed without merging**